make.inc.macosx_gcc-10

```diff
@@ -0,0 +1,54 @@
+# Makefile variable overrides for Mac OSX compilation with GCC v.10.*
+#
+# Use this if you'll need to link against gfortran.
+#
+# Copy this file to make.inc, and if needed edit for your setup.
+#
+# Don't forget also to adjust g++ to g++-10 at start of setup.py
+#
+# Barnett 10/27/18. Input from Yu-Hsuan Shih, Amit Moskovich.
+# modified 12/06/2020
```

Comment:
feel free to add your name in the old-style way.

Reply:
I just changed gcc-8 to gcc-10.